### PR TITLE
Location Service 

### PR DESCRIPTION
**Pull request by hadric**
Added a Location Service in the samples section.

added a readme file - https://github.com/hadric/cowj/blob/main/README.md

### CODE REVIEW

File: app/samples/LocatorSerivce/distance.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+
+// request body
+input = req.attribute("_body");
+person1 = input.get("person_id_1");
+person2 = input.get("person_id_2");
+
+// utilities
+function getDataFromSql(person, jdbc, redis) {
```
Comment (nmondal) [inline app/samples/LocatorSerivce/distance.js:21]: Not required. There is literally a select() in the JDBCWrapper.
It is also being used in tests.

File: app/samples/LocatorSerivce/locations.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+data = [];
+try {
+
+if(checkIfUserRegistered(person, jdbc) != '') {
+dataTemp = getDataFromSql(person, startTime, endTime, jdbc);
+if(dataTemp != '') {
+data = dataTemp;
+}
+} else {
+Test.expect(false, 'User not found', 404 )
```
Comment (nmondal) [inline app/samples/LocatorSerivce/locations.js:57]: The idea of expect was to avoid if/else.
Also that is why Test.panic() exists.

**Review by nmondal — CHANGES_REQUESTED** [review]
Check the comments!

File: app/samples/LocatorSerivce/distance.js (hunk 1 above)
Comment (hadric) [reply app/samples/LocatorSerivce/distance.js:21]: Yes, i checked the tests, it works for Derby.
when i use mysql and try jdbc.select(findPerson), i am getting below error
InternalError: Can't find method cowj.plugins.JDBCWrapper.select(org.mozilla.javascript.ConsString). (eval#15)

File: app/samples/LocatorSerivce/locations.js (hunk 2 above)
Comment (hadric) [reply app/samples/LocatorSerivce/locations.js]: made the changes .Please check now

File: app/samples/LocatorSerivce/distance.js (hunk 1 above)
Comment (hadric) [reply app/samples/LocatorSerivce/distance.js:21]: again looks like an issue with mozilla rhino

Comment (hadric) [reply app/samples/LocatorSerivce/distance.js:21]: made the changes. Please check.
i was not passing all the params in the select method .

**Comment by nmondal** [comment]
This was a cool one, great effort. Some interesting bugs were found.
No need to add this to the repo, should be here for historical reason.

**Pull request closed without merging**